src/layout/theme.liquid

```diff
@@ -56,11 +56,21 @@
   <!--[if (gt IE 9)|!(IE)]><!--><script src="{{ 'theme.js' | asset_url }}" defer="defer"></script><!--<![endif]-->
   <!--[if lt IE 9]><script src="{{ 'theme.js' | asset_url }}"></script><![endif]-->
 
+  <script src="{{ 'cookie_disclaimer.js' | asset_url }}" defer="defer"></script>
```

Comment:
@gts-gi instead of adding another JS file, we concat this to theme.js using ES6 requires - it keeps it tidier as it's theme specific